nautobot/core/jobs/bulk_actions.py

```diff
@@ -108,7 +108,7 @@ def _update_objects(self, model, form, filter_query_params, edit_all, nullified_
                         if form.cleaned_data[field_name]:
                             getattr(obj, field_name).set(form.cleaned_data[field_name])
                     # Normal fields
-                    elif form.cleaned_data[field_name] not in (None, ""):
+                    elif form.cleaned_data[field_name] not in (None, "", []):
```

Comment:
If this change fixes a bug that end users could encounter, please document it in a `changes/6962.fixed` file.